package.json

```diff
@@ -30,53 +31,38 @@
     "videojs-plugin"
   ],
   "dependencies": {
-    "global": "^4.3.2"
+    "@babel/runtime": "^7.5.5"
   },
   "devDependencies": {
-    "conventional-changelog-cli": "^2.0.1",
```

Comment:
all of these are part of the generator-helpers?

Reply:
yeah